examples/logback/README.md

```diff
@@ -0,0 +1,9 @@
+# Bugsnag Logback Java Example
+
+A Spring Boot example application to show how to use the Bugsnag Logback appender in a Spring based Java application.
+
+- Run the app
+
+    ```shell
+    ../../gradlew clean bootRun
+    ```
```

Comment:
We should add instructions on how to change the API key

Reply:
added